internal/util/searchreplace/searchreplace.go

```diff
@@ -0,0 +1,174 @@
+package searchreplace
```

Comment:
Putting this inside a `util` package seems unnecessary. Lets just add the searchreplace package directly under `internal`.

Reply:
As commented above, this is the pattern we are following for other commands. We should do it as a clean-up in a single different PR across all commands.

Reply:
I think we should try to move towards the more common way of structuring go projects, but I'm ok with doing that separately.